test/ibmq/test_proxies.py

```diff
@@ -15,20 +15,18 @@
 """Tests for the AuthClient and VersionClient proxy support."""
 
 
-from unittest import skipIf
-import urllib
 import subprocess
-import sys
+import urllib
+from sys import version_info as python_version
```

Comment:
Minor (and style-related), but in general we avoid using an alias when importing unless strongly justified (ie. in case of conflicts, etc). The rationale is to avoid one extra layer of indirection - on a global scale, when re-reading the code it is arguably faster to mentally parse just `version_info` or `sys.version_info` directly, specially if the reader has played with that part of the standard library.